vision.md

```diff
@@ -67,28 +67,50 @@ also by attaching custom functionality with minimum effort.
 
 ## Users and Personas
 
-Within the different target segments, we are addressing the following personas
-with thin-edge.io:
-
-**(IoT) Solution Developer/ Solution Architect**: Background consisting of
-Python, Java, JS, Angular, Kubernetes, Cloud Platforms
-- Responsible for :
-    - Implementing and maintaining the end-to-end IoT solution
-    - Often juggling multiple initiatives covering a broad range of technology
-      stacks in addition to implement and maintaining solutions.
+We define the following personas:
+
+- "End User"
+    - Person using a binary that has been built with the thin-edge.io framework
+      as well as a finished and working configuration
+
+- "System Integrator"
+    - Person using a binary that has been built with the thin-edge.io framework
+      and crafts a configuration suited to solve their problems
```

Comment:
I agree with Didier suggestion with one small change I would name Fleet Administrator as standard System Operator

Reply:
I do think that the "End User" needs to be in the picture. This person is interacting with IoT devices using an application built on top of thin-edge. Many requirements make sense only because there are such end users. They main requirement might be to be oblivious of thin-edge; but this will not come without an explicit attention from us.